### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by BradLugo**

### CODE REVIEW

File: crates/validation/macros/src/lib.rs [hunk 1]
```diff
@@ -108,6 +108,7 @@ pub fn validated_type(input: TokenStream) -> TokenStream {
                 #validated_type_ident,
                 ::validation::ValidationError
             > {
+                panic!("here");
```
Comment (dmah42) [inline crates/validation/macros/src/lib.rs:111]: is this debugging code? :)

Comment (BradLugo) [reply crates/validation/macros/src/lib.rs:111]: Yeah 😞 There may be a few instances of this before the PR is ready. Most of the advice I found when searching how to debug macro issues was along the lines of "just add debug statements until you figure it out." 

If anyone has a better way to debug macros, please let me know.

**Review by dmah42 — DISMISSED** [review]

File: crates/validation/macros/src/lib.rs (hunk 1 above)
Comment (dmah42) [reply crates/validation/macros/src/lib.rs]: no, that's about how i do it.

no worries about giving up on some of the packages. we can do those one-by-one.

**Comment by dmah42** [comment]
merged in some other PRs and i think i got the merge right here.  hopefully it doesn't make things harder for you.

**Comment by dmah42** [comment]
manually merged Cargo.lock :/

**Comment by dmah42** [comment]
we just did a big dependency update in #496 so i'm going to close this in deference to that.

**Pull request closed without merging**